packages/sdk/README.md

```diff
@@ -102,53 +102,36 @@ Executes a Vincent Tool with the provided parameters.
 ### Usage
 
 ### Authentication
-```typescript
-import { NextFunction, Request, Response } from 'express';
+A basic Express authentication middleware factory function is provided with the SDK.
```

Comment:
Replaced explicit express-specific code with references to the methods that provide that functionality, and the interface that consumers can use to work with it in a type-safe way.